doc/v1/docs/superadmin_metric_templates.md

```diff
@@ -24,7 +24,56 @@ Mandatory fields in case of active metric template:
 * **Probe** - probe name and version. This field is autocomplete, user must choose from available probes;
 * **Type** - type (active or passive; active in this example);
 * **Probe executable** - executable file;
-* **Config** - those defined 5 config fields are mandatory for active metrics.
+* **Config** 
+    * **maxCheckAttempts**,
+    * **timeout**,
+    * **path**,
+    * **interval**,
+    * **retryInterval**.
+
+#### Attributes
+
+Attributes are part of metric template definition. They are used to pass values as arguments to metric command (e.g. TCP port on which service is listening, SE path, CE queue...). 
+
+Description of attributes is given in the table below.
+
+| Attribute | Description |
```

Comment:
Add:
URL - URL extracted topology database (e.g. GOCDB attribute URL)
\<serviceType\>_URL - URL extracted topology database (e.g. GOCDB attribute URL); serviceType prefix is used in order to avoid collision in case of multiple serviceTypes on a same host
PORT - Port extracted from URL
PATH - Path extracted from URL
SSL - Attribute is set if URL scheme is https://
HOST_NAME - Hostname of the monitored box. Do not use this attribute for parameter -H because it is set automatically.
TOP_BDII - Hostname of Top BDII used for generating configuration.
KEYSTORE - Location of Java keystore with host certificate. Default value is '/etc/nagios/globus/keystore.jks'.
TRUSTSTORE - Location of Java truststore with all IGTF CA bundle. Default value is '/etc/nagios/globus/truststore.ts'.